client/my-sites/stats/activity-log/index.jsx

```diff
@@ -595,67 +615,69 @@ class ActivityLog extends Component {
+								end
+									.clone()
+									.utc()
+									.startOf( 'day' )
+							);
+
+							switch ( type ) {
+								case 'empty-day':
+									return (
+										<div key={ start.format() } className="activity-log__empty-day">
+											<div className="activity-log__empty-day-title">
+												{ start.format( 'LL' ) }
+												{ isToday && ` \u2014 ${ translate( 'Today' ) }` }
 											</div>
-										);
-
-									case 'empty-range':
-										return (
-											<div key={ start.format( 'LL' ) } className="activity-log__empty-day">
-												<div className="activity-log__empty-day-title">
-													{ `${ start.format( 'LL' ) } - ${ end.format( 'LL' ) }` }
-													{ isToday && ` \u2014 ${ translate( 'Today' ) }` }
-												</div>
-												<div className="activity-log__empty-day-events">
-													{ translate( 'No activity' ) }
-												</div>
+											<div className="activity-log__empty-day-events">
+												{ translate( 'No activity' ) }
 											</div>
-										);
-
-									case 'non-empty-day':
-										return (
-											<ActivityLogDay
-												key={ start.format() }
-												applySiteOffset={ this.applySiteOffset }
-												requestedRestoreActivityId={ requestedRestoreActivityId }
-												requestedBackupId={ requestedBackupId }
-												restoreConfirmDialog={ restoreConfirmDialog }
-												backupConfirmDialog={ backupConfirmDialog }
-												disableRestore={ disableRestore }
-												disableBackup={ disableBackup }
-												hideRestore={ ! rewindEnabledByConfig || ! isPressable }
-												isRewindActive={ isRewindActive }
-												logs={ events }
-												requestDialog={ this.handleRequestDialog }
-												closeDialog={ this.handleCloseDialog }
-												siteId={ siteId }
-												tsEndOfSiteDay={ start.valueOf() }
-												isToday={ isToday }
-											/>
-										);
-								}
-							} ) }
+										</div>
+									);
+
+								case 'empty-range':
+									return (
+										<div key={ start.format( 'LL' ) } className="activity-log__empty-day">
+											<div className="activity-log__empty-day-title">
+												{ `${ start.format( 'LL' ) } - ${ end.format( 'LL' ) }` }
+												{ isToday && ` \u2014 ${ translate( 'Today' ) }` }
+											</div>
+											<div className="activity-log__empty-day-events">
+												{ translate( 'No activity' ) }
+											</div>
+										</div>
+									);
+
+								case 'non-empty-day':
+									return (
+										<ActivityLogDay
+											key={ start.format() }
+											applySiteOffset={ this.applySiteOffset }
+											requestedRestoreActivityId={ requestedRestoreActivityId }
+											requestedBackupId={ requestedBackupId }
+											restoreConfirmDialog={ restoreConfirmDialog }
+											backupConfirmDialog={ backupConfirmDialog }
+											disableRestore={ disableRestore }
+											disableBackup={ disableBackup }
```

Comment:
Maybe we can update this to have something like `disableAction={ disableAction }` where for example if a restore is on progress we have
```
disableAction = {
    restore: true,
    backup: false,
    transfer: false
}
```
so we pass down only one prop, but this can be done in a later PR. We could do something similar for handlers.